### PR TITLE
Uplift minikube to 1.25.1 and comment out few renovate updates

### PR DESCRIPTION
**Pull request by kashifest**
If Kubernetes version is not 1.21.2, minikube will be using 1.25.1 version.

We are commenting out kind node image and minikube version upgrade because of conflicting versions used in for different CAPM3 api versions.

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -46,12 +46,12 @@
...
-       "matchStrings": ["MINIKUBE_VERSION:-\"(?<currentValue>.*?)\"}"],
-       "depNameTemplate": "kubernetes/minikube",
-       "datasourceTemplate": "github-releases"
-     },
+    //  {
+    //    "fileMatch": ["^lib/common.sh$"],
+    //    "matchStrings": ["MINIKUBE_VERSION:-\"(?<currentValue>.*?)\"}"],
+    //    "depNameTemplate": "kubernetes/minikube",
+    //    "datasourceTemplate": "github-releases"
+    //  },
```
Comment (furkatgofurov7) [inline renovate.json:54]: Once we drop CAPM3 v1a4, we can uncomment this again and remove if/else condition from lib/common.sh

**Review by furkatgofurov7 — APPROVED** [review]
/approve

This replaces https://github.com/metal3-io/metal3-dev-env/pull/854

**Review by lentzi90 — COMMENTED** [review]
Not sure if it is actually possible to comment out things in `renovate.json`. Normally json files do not accept comments. Do you know if renovate understands this syntax?

**Review by Sunnatillo — APPROVED** [review]
/lgtm

**Comment by kashifest** [comment]
> Not sure if it is actually possible to comment out things in `renovate.json`. Normally json files do not accept comments. Do you know if renovate understands this syntax?

Not sure either. My xcode added this so I was thinking it should be fine. Whats your opinion on this.

**Comment by kashifest** [comment]
> > Not sure if it is actually possible to comment out things in `renovate.json`. Normally json files do not accept comments. Do you know if renovate understands this syntax?
> 
> Not sure either. My xcode added this so I was thinking it should be fine. Whats your opinion on this.

I have removed it. We can always add those when we are ready to @lentzi90

**Review by lentzi90 — APPROVED** [review]
/lgtm

**Review by smoshiur1237 — COMMENTED** [review]
/lgtm

**Review by Sunnatillo — APPROVED** [review]

**Pull request merged**